geometries/moonboresight/moonboresight_algorithm.py

```diff
@@ -70,3 +70,40 @@ def moonboresight_instr(utc, target, instr):
     sb_deg = sp.convrt(sb_rad, 'RADIANS', 'DEGREES')
 
     return sb_deg
+
```

Comment:
The following code is a simpler version that I recommend to replace the versions above.